tsconfig.json

```diff
@@ -15,8 +15,8 @@
 		"newLine": "lf",
 		"stripInternal": true,
 		"noImplicitReturns": true,
-		"noUnusedLocals": true,
-		"noUnusedParameters": true,
+		"noUnusedLocals": false,
+		"noUnusedParameters": false,
```

Comment:
IMO this is a linter concern, not a compiler one. It's duplicated my the `no-unused-variables` rule, and better via eslint because:

- if we do need to suppress it in one-off cases, it's safer to disable the specific rule with eslint rather than `// @ts-ignore`
- the eslint rule is more configurable. We can allow certain regexes to establish conventions
- ts-jest will refuse to run if there are compiler errors. This can be turned off but it's a useful feature. But when developing it's or pretty common to run tests with tempors unused variables. They won't get into master anyway because of the lint rule.

Reply:
I'm convinced!